<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <style type="text/css"> .section {background:#ffffdd;margin:3px;border-style:solid;border-width:0.5px;}.table {background:#ffddff;margin:3px;border-style:solid;border-width:0.5px;}.figure {background:#ddffff;margin:3px;border-style:solid;border-width:0.5px;}.image {background:#dddddd;margin:3px;border-style:solid;border-width:0.5px;font-size:30;}</style>
 </head>
 <body>
  <div class="page">
   <div class="running">
    <div class="subSection" title="RE S E AR CH ART I CL E Open Access">
     <span style="fill:#ffffff;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:13.0;font-style:normal;font-weight:normal;" x="60.718" y="110.728">RE S E AR CH ART I CL E Open Access</span>
     <img src="./images/page.1.1.png" width="50%"/>
     <span x="484.242" y="120.722" style="font-size:10pt;" class="image">image.1.1[22*22]</span>
    </div>
   </div>
   <p>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:11.0;font-style:normal;font-weight:normal;" x="56.693" y="172.693">Sanjarbek Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:11.0;font-style:normal;font-weight:normal;" x="159.25" y="172.693">, Sergey Shmakov</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.33;font-style:normal;font-weight:normal;" x="233.461" y="167.818">1,2</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:11.0;font-style:normal;font-weight:normal;" x="242.249" y="172.693">, Yuri I. Wolf</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:11.0;font-style:normal;font-weight:normal;" x="295.937" y="172.693">, Michael P. Terns</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:11.0;font-style:normal;font-weight:normal;" x="372.189" y="172.693">, Kira S. Makarova</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:11.0;font-style:normal;font-weight:normal;" x="56.693" y="186.696">and Eugene V. Koonin</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.33;font-style:normal;font-weight:normal;" x="149.896" y="181.821">1*</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="243.219">Background:</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="120.643" y="243.219">The Cas4 family endonuclease is a component of the adaptation module in many variants of CRISPR-Cas</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="255.238">adaptive immunity systems. Unlike most of the other Cas proteins, Cas4 is often encoded outside CRISPR-</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="477.243" y="255.238">cas</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="492.096" y="255.238">loci</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="267.2">(solo-Cas4) and is also found in mobile genetic elements (MGE-Cas4).</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="98.136" y="282.224">As part of our ongoing investigation of CRISPR-Cas evolution, we explored the phylogenomics of the Cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="294.244">family. About 90% of the archaeal genomes encode Cas4 compared to only about 20% of the bacterial genomes.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="306.206">Many archaea encode both the CRISPR-associated form (CAS-Cas4) and solo-Cas4, whereas in bacteria, this</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="318.225">combination is extremely rare. The solo-</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="220.366" y="318.225">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="239.698" y="318.225">genes are over-represented in environmental bacteria and archaea with</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="330.244">small genomes that typically lack CRISPR-Cas, suggesting that Cas4 could perform uncharacterized defense or repair</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="342.206">functions in these microbes. Phylogenomic analysis indicates that both the CRISPR-associated</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="432.285" y="342.206">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="451.617" y="342.206">genes are often</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="354.225">transferred horizontally but almost exclusively, as part of the adaptation module. The evolutionary integrity of the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="366.244">adaptation module sharply contrasts the rampant shuffling of CRISPR-</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="337.04" y="366.244">cas</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="351.837" y="366.244">modules whereby a given variant of the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="378.206">adaptation module can combine with virtually any effector module. The solo-</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="367.825" y="378.206">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="387.157" y="378.206">genes evolve primarily via vertical</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="390.226">inheritance and are subject only to occasional horizontal transfer. The selection pressure on</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="423.781" y="390.226">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="443.113" y="390.226">genes does not</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="402.245">substantially differ between CAS-Cas4 and solo-</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="249.846" y="402.245">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="266.458" y="402.245">, and is close to the genomic median. Thus,</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="439.768" y="402.245">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="459.101" y="402.245">genes, similarly to</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="414.207">cas1</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="82.545" y="414.207">and</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="100.176" y="414.207">cas2</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="116.787" y="414.207">, evolve similarly to</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:10.0;font-style:normal;font-weight:normal;" x="193.89" y="414.207">‘</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="195.421" y="414.207">regular</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:10.0;font-style:normal;font-weight:normal;" x="222.633" y="414.207">’</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="226.942" y="414.207">microbial genes involved in various cellular functions, showing no evidence</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="426.226">of direct involvement in virus-host arms races. A notable feature of the Cas4 family evolution is the frequent</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="438.245">recruitment of</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="121.947" y="438.245">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="141.279" y="438.245">genes by various mobile genetic elements (MGE), particularly, archaeal viruses. The functions of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="450.207">Cas4 in these elements are unknown and potentially might involve anti-defense roles.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="118.942" y="465.231">Unlike most of the other Cas proteins, Cas4 family members are as often encoded by stand-alone genes</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="477.193">as they are incorporated in CRISPR-Cas systems. In addition,</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:10.0;font-style:normal;font-weight:normal;" x="298.659" y="477.193">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="317.991" y="477.193">genes were repeatedly recruited by MGE, perhaps, for</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="489.213">anti-defense functions. Experimental characterization of the solo and MGE-encoded Cas4 nucleases is expected to</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="501.232">reveal currently uncharacterized defense and anti-defense systems and their interactions with CRISPR-Cas systems.</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="56.693" y="531.672">Background</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="543.691">Cas4 is one of the core CRISPR-associated (Cas) proteins,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="555.653">which is implicated in the adaptation phase in several</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="567.672">subtypes of CRISPR-Cas systems [1</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="199.728" y="567.672">–</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="205.454" y="567.672">4]. During this phase</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="579.69">of the CRISPR response, a segment of invading DNA</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="591.652">(protospacer), usually from a virus or plasmid, is selected,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="603.671">typically, based on the presence of a protospacer adjacent</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="615.689">motif (PAM) [5, 6]. The protospacer then is incorporated</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="627.651">by the Cas1-Cas2 adaptation complex into a CRISPR array,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="639.67">most often, next to the first, 5</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="178.582" y="639.67">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="182.21" y="639.67">-terminal repeat. The array</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="531.67">is transcribed into a single, long pre-crRNA, which is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="531.67">subsequently processed into mature CRISPR (cr)RNAs and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="543.688">is incorporated into the effector complex or a single ef-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="555.65">fector protein, which scans invading DNA and, once a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="567.669">match is found, recruits nuclease domain-containing Cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="579.687">proteins to cleave the target DNA [7</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="453.824" y="579.687">–</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="459.607" y="579.687">10]. So far the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="521.459" y="579.687">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="591.649">gene has been identified in the subtypes I-A, B, C, D, U of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="603.668">Class 1 CRISPR-Cas systems and subtypes II-B and V-A, B,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="615.686">E of Class 2. In each of the respective loci, the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="491.185" y="615.686">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="511.198" y="615.686">gene is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="627.648">adjacent to</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="353.024" y="627.648">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="374.001" y="627.648">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="392.539" y="627.648">cas2</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="409.433" y="627.648">, and in some cases, fused with</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="639.667">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="321.673" y="639.667">, which is compatible with the involvement of Cas4 in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="651.685">adaptation [11, 12]. Furthermore, it has been shown that in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="663.647">subtype I-A loci,</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="375.815" y="663.647">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="392.766" y="663.647">,</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="398.718" y="663.647">cas2</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="419.638" y="663.647">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="438.063" y="663.647">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="458.983" y="663.647">genes form a single</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="675.666">operon [1], and for subtype I-B, the requirement of Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="687.684">for adaptation has been demonstrated in a direct experi-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="699.646">ment [3]. A recent study on the cas gene requirement for</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="680.435">* Correspondence: koonin@ncbi.nlm.nih.gov</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="59.357" y="689.449">National Center for Biotechnology Information, National Institutes of Health,</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="698.463">Bethesda, MD, USA</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="707.477">Full list of author information is available at the end of the article</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.0;font-style:normal;font-weight:normal;" x="180.453" y="724.428">© The Author(s). 2017</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.0;font-style:normal;font-weight:normal;" x="240.774" y="724.428">Open Access</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.0;font-style:normal;font-weight:normal;" x="279.553" y="724.428">This article is distributed under the terms of the Creative Commons Attribution 4.0</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.0;font-style:normal;font-weight:normal;" x="180.453" y="732.422">International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.0;font-style:normal;font-weight:normal;" x="180.453" y="740.416">reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.0;font-style:normal;font-weight:normal;" x="180.453" y="748.41">the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.0;font-style:normal;font-weight:normal;" x="180.453" y="756.404">(http://creativecommons.org/publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.139">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.139">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.139">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="58.116">DOI 10.1186/s12862-017-1081-1</span>
   </p>
   <div class="section" title="Phylogenomics of Cas4 family nucleases">
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:24.0;font-style:normal;font-weight:normal;" x="56.693" y="148.712">Phylogenomics of Cas4 family nucleases</span>
   </div>
   <div class="section" title="Abstract">
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="63.213" y="225.191">Abstract</span>
   </div>
   <div class="section" title="Results: ">
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="282.224">Results:</span>
   </div>
   <div class="section" title="Conclusions: ">
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:10.0;font-style:normal;font-weight:normal;" x="63.213" y="465.231">Conclusions:</span>
    <img src="./images/page.1.2.png" width="50%"/>
    <span x="373.493" y="42.696" style="font-size:10pt;" class="image">image.1.2[165*30]</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="104.492">adaptation by the type I-A system of the crenarchaeon</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="274.733" y="104.492">Sul-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="116.453">folobus islandicus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="133.908" y="116.453">has shown that two paralogous</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="273.429" y="116.453">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="128.472">genes (one denoted</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="145.133" y="128.472">csa1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="162.084" y="128.472">) in the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="202.733" y="128.472">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="220.875" y="128.472">operon are both</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="140.49">essential, suggesting that these proteins could form a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="152.452">complex in which the two paralogs would perform unique</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="164.471">functions [13].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="176.489">Cas4 is a DNA exonuclease that contains a nuclease</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="188.451">domain and a Fe-S cluster-binding module [2, 14</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="271.615" y="188.451">–</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="277.567" y="188.451">16]</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="200.47">and is homologous to the nuclease moieties of well-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="212.488">characterized proteins involved in recombination and re-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="224.45">pair in bacteria (RecB, AddB) and eukaryotes (Dna2)</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="236.469">[17</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="69.675" y="236.469">–</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="75.627" y="236.469">21]. The Cas4 nuclease belongs to the expansive</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="248.487">PD-DEXK phosphodiesterase superfamily, named after</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="260.449">the conserved catalytic motif present in most members</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="272.468">[21]. Similar to the AddB nuclease, Cas4 is thought to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="284.486">form recombinogenic 3</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="153.41" y="284.486">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="160.44" y="284.486">ssDNA overhangs in the proto-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="296.448">spacers, thus facilitating their subsequent incorporation</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="308.467">into the CRISPR array as dsDNA spacers [16]. In the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="320.485">structures that have been solved for Cas4 proteins from</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="332.447">Pyrobaculum calidifontis</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="166.789" y="332.447">(Pcal_0546; pdb: 4R5Q) [15]</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="344.466">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="77.441" y="344.466">Sulfolobus solfataricus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="176.767" y="344.466">(SSO0001, pdb: 4IC1) [2],</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="356.484">Pcal_0546 is a monomer, whereas SSO0001 forms a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="368.446">toroidal decamer composed of 5 tightly packed dimers.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="380.465">The Pcal_0546 protein has been shown to exhibit a metal-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="392.483">dependent 5</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="108.339" y="392.483">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="116.787" y="392.483">to 3</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="134.532" y="392.483">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="142.979" y="392.483">exonuclease activity against ssDNA</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="404.445">substrates [15], whereas the Cas4 protein SSO1391 from</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="416.464">S. solfataricus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="115.143" y="416.464">has been reported to cleave ssDNA in both</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="428.482">the 5</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="76.933" y="428.482">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="83.396" y="428.482">to 3</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="99.1" y="428.482">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="105.563" y="428.482">and the 3</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="143.377" y="428.482">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="149.84" y="428.482">to 5</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="165.544" y="428.482">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="172.007" y="428.482">directions [2]. However, in an</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="440.444">independent study, only the 5</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="176.26" y="440.444">′</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="188.221" y="440.444">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="194.798" y="440.444">exonuclease activity has</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="452.462">been detected for SSO1391 as well [16].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.689" y="464.481">Although the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="119.793" y="464.481">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="139.806" y="464.481">gene is usually located next to</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="262.035" y="464.481">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="282.047" y="464.481">or</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="476.443">cas2</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="77.557" y="476.443">genes and, in some CRISPR-Cas systems, is fused to</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="488.461">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="76.026" y="488.461">(eg. subtypes I-U and V-B), many bacterial and archaeal</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="500.48">genomes encompass an additional</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="190.149" y="500.48">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="209.311" y="500.48">gene that is not asso-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="512.442">ciated with CRISPR-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="134.986" y="512.442">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="149.557" y="512.442">loci (hereinafter, solo-Cas4). Notably,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="524.46">solo-cas4 genes are present in many archaeal and bacterial</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="536.479">genomes that lack CRISPR-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="162.483" y="536.479">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="176.996" y="536.479">loci [11].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.689" y="548.441">The recent discovery of Cas4 homologs encoded in bac-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="560.459">teriophage genomes has added a new twist to the func-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="572.478">tional repertoire of the Cas4 family. It has been shown</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="584.44">that a</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="88.273" y="584.44">Campylobacter</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="154.661" y="584.44">phage that encodes a Cas4-like</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="596.458">protein, employs this protein to stimulate acquisition of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="608.42">host-derived spacers by the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="183.517" y="608.42">Campylobacter</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="250.699" y="608.42">type II-C</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="620.439">CRISPR-Cas system (which lacks</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="200.582" y="620.439">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="217.93" y="620.439">). The phage ap-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="632.457">pears to use this mechanism to evade host immunity via a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="644.419">mechanism that remains to be characterized [22, 23].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.69" y="656.438">Cas4 homologs have been identified also in archaeal vi-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="668.456">ruses [24] and casposons, self-synthesizing transposons</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="680.418">that employ Cas1 homologs as recombinases and are</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="692.437">thought to be the ancestors of the CRISPR-Cas adaptation</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="704.455">modules [25, 26]. One archaeal virus, Thermoproteus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="716.417">tenax virus 1 (TTV1), encodes an inactivated derivative of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="728.435">a Cas4-like nuclease that became a component of the viral</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.696" y="104.476">nucleocapsid [24].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.72" y="104.476">Despite the current keen interest in the mechanisms,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.726" y="116.438">diversity and evolution of the CRISPR-Cas systems, no</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.726" y="128.456">comprehensive analysis of the Cas4 family has been re-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="140.492">ported so far. The aim of this work is to present such an</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="152.453">analysis with the focus on</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="415.842" y="152.453">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="436.932" y="152.453">gene neighborhoods and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="164.472">comparison of the (predicted) functions and evolution-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="176.49">ary regimes of CRISPR-associated Cas4 and solo-Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="188.452">subfamilies.</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="304.724" y="212.492">Results and discussion</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="304.724" y="224.454">Phylogenomic analysis of the Cas4 protein family</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="236.473">As reported previously, the majority of the Cas4 proteins</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="248.491">belong to two families, namely, COG1468 and COG4343;</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="260.453">the latter is specific for subtype I-A CRISPR-Cas systems</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="272.472">and is also known as Csa1 (these two COGs correspond</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="284.49">to pfam01930/DUF83 and pfam06023/ DUF911, respect-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="296.452">ively) [11]. In addition, Cas4-like nucleases of pfam12705/</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="308.47">PDDEXK_1 are occasionally found in association with</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="525.996" y="308.47">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="320.489">loci and have been shown to affect spacer adaptation by</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="332.451">certain CRISPR-Cas systems [22, 23]. In the present ana-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="344.469">lysis, we focused primarily on these three major families</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="356.488">of Cas4 homologs and disregarded a few other, less com-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="368.45">mon PD-DExK families that also have been reported in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="380.468">the vicinity of several CRISPR-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="427.124" y="380.468">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="442.261" y="380.468">loci [11].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.718" y="392.487">To collect protein sequences of the Cas4 family, we</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="404.449">screened 48,599 prokaryotic genomes (Genbank, March</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="416.467">2016) using PSI-BLAST search [27] with the sequence</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="428.486">profiles related to the three families of Cas4 homologs.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="440.448">The results of this search were filtered to exclude se-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="452.466">quences that produced better scores to other profiles in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="464.485">the CDD database (See Material and Methods for de-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="476.447">tails). Additionally, we searched 2996 phage genomes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="488.465">from the PHANTOM database [28]. In total, 7060 pro-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="500.484">tein sequences were retrieved, of which 883 were from</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="512.446">complete bacterial and archaeal genomes, and 272 were</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="524.464">from bacterial or archaeal viruses.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.718" y="536.483">Inspection of the multiple alignment of Cas4 proteins</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="548.445">(see Materials and Methods) shows conservation of the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="560.463">catalytic motifs of the PD-DEXK phosphodiesterase</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="572.482">superfamily, suggesting that all Cas4 proteins are active</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="584.444">nucleases. All members of the Cas4 family contain a Fe-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="596.462">S-cluster-binding module with 4 conserved cysteines. It</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="608.481">has been shown that different Cas4 protein bind either</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="620.442">[2Fe-2S] or [4Fe-4S] clusters [2, 15]; however, no</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="307.559" y="632.461">consistent difference in the patterns of sequence conser-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="644.48">vation was detected between proteins binding these dis-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="656.441">tinct ligands.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.718" y="668.46">We then constructed a sequence similarity dendro-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="680.479">gram (see Materials and Method for details) for the 7060</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="692.44">identified Cas4 family proteins and mapped the gene</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="704.459">context to each branch, including assignments to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="716.478">CRISPR-Cas subtypes (Additional file 1: Figure S1). As</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="728.439">expected, the sequences that belong to distinct pfam en-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="740.458">tries and COGs do not overlap in the tree, and pfam12705</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.139">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.139">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.139">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.061" y="48.139">Page 2 of 14</span>
   </p>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="104.492">contained no major branches associated with CRISPR-Cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="116.453">systems; thus, hereinafter, we only briefly discuss this fam-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="128.472">ily. The remaining Cas4 homologs could be classified into</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="140.49">three major groups based on the genomic context: 1)</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="152.452">CRISPR-Cas-associated Cas4 (CAS-Cas4), 2) Cas4 associ-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="164.471">ated with mobile genetic elements (MGE-Cas4), and 3)</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="176.489">solo-Cas4 (i.e.</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="121.947" y="176.489">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="145.304" y="176.489">genes located outside functionally</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="188.451">characterized genomic contexts). The CAS-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="232.78" y="188.451">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="253.076" y="188.451">genes are</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="200.47">embedded in several typical arrangements of the adapta-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="212.488">tion module that also include the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="202.96" y="212.488">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="224.787" y="212.488">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="244.062" y="212.488">cas2</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="265.945" y="212.488">genes,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.45">and are often shared between several CRISPR-Cas sub-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="236.469">types (Fig. 1). As reported previously, Cas1 and Cas4 pro-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="248.487">teins are fused in some systems (I-B, I-U and V-A),</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="260.449">whereas I-A is the only CRISPR-Cas subtype that typically</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="272.468">contains two</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="110.834" y="272.468">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="128.125" y="272.468">-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="131.641" y="272.468">like</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="148.251" y="272.468">genes from two distinct subfamilies</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="284.486">within the adaptation module (Fig. 1). One of the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="296.448">conserved arrangements shared by some of I-B and I-D</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="308.467">systems also includes the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="158.286" y="308.467">cas6</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="178.129" y="308.467">gene, which is not known to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="320.485">be involved in adaptation, but has been identified in several</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="332.447">fusions with Cas1 and reverse transcriptase domains in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="344.466">some type III CRISPR-Cas systems (Fig. 1) [29, 30].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.688" y="356.484">We further compared the phyletic distributions of the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="368.446">CAS-Cas4 and solo-Cas4 in complete microbial ge-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="380.465">nomes (Fig. 2a). This analysis revealed pronounced dif-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="392.483">ferences between the distributions of the Cas4 family</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="404.445">members in archaea and bacteria. The Cas4 family is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="416.464">represented in about 90% of the archaeal genomes, and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="104.484">about 50% of these encode both CAS-Cas4 and solo-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="104.484">Cas4 (Additional file 4: Information File S2). We further</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="116.445">compared the representation of solo-Cas4 and distinct</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="128.464">CAS-Cas4 contexts for the complete genome set,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="140.483">including draft genomes, counting one strain per species</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="152.444">for each category to avoid sequencing bias (Fig. 2b).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="164.463">Bacteria show a patchy distribution of the Cas4 family</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="176.482">genes, with a smaller fraction of solo-Cas4 compared to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="188.443">archaea (Fig. 2b). Numerous archaeal genomes encode</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="200.462">both CAS-Cas4 and solo-Cas4 but the frequency of this</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="212.481">combination is close to the product of individual fre-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="224.442">quencies of each form, thus showing no evidence of a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="236.461">functional connection. Notably, most members of the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="248.479">DPANN superphylum (a recently discovered major,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="260.441">apparently monophyletic group of archaea that includes</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="272.46">Aenigmarchaeota, Diapherotrites, Micrarchaeota, Nano-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="284.478">haloarchaeota, Pacearchaeota, Parvarchaeota</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="509.838" y="284.478">, and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="296.44">Woesearchaeota</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="370.998" y="296.44">), largely symbiotic archaea with small</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="308.459">genomes [31, 32], encode solo-Cas4, but not CAS-Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="320.477">(Additional file 7: Table S1). Within the entire genomic</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="332.439">set, which includes draft genomes, CAS-Cas4 is most</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="344.458">often present in type I-B and I-C CRISPR-Cas loci (Fig.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="356.476">2b). We also observed many cases of CAS-Cas4 being a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="368.438">component of stand-alone (not associated with effector</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="380.457">modules) adaptation modules, which are present in about</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="392.475">12% of the archaea and 4% of the bacteria that possess at</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="404.437">least one</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="343.332" y="404.437">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="363.685" y="404.437">gene (Fig. 2b). Conceivably, the high preva-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="416.456">lence of Cas4 in archaea is due to the dominance of I-B</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="428.474">and I-C systems as opposed to I-E and I-F that are most</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.869" y="694.211">Typical organization of adaptation modules that include</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:8.0;font-style:normal;font-weight:normal;" x="267.306" y="694.211">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="283.237" y="694.211">genes. Most common organizations of adaptation modules containing</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:8.0;font-style:normal;font-weight:normal;" x="512.502" y="694.211">cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.703" y="704.189">gene are shown. Genes are shown by block arrows according to the transcription direction and are not to scale. Homologous genes are color-</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.703" y="714.224">coded and identified by a systematic name and a legacy name (indicated by an asterisk). Cas4 family genes are indicated by thick outline.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.703" y="724.202">CRISPR-Cas system subtypes found in association with each module are indicated. Notably, the adaptation modules of the I-B and I-D loci include</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:8.0;font-style:normal;font-weight:normal;" x="62.703" y="734.236">cas6</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="78.634" y="734.236">genes</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.694" y="48.141">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.863" y="48.141">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.42" y="48.141">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.062" y="48.141">Page 3 of 14</span>
   </p>
   <div class="figure" title="Fig. 1 ">
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="694.211">Fig. 1</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="656.454">common in bacteria and lack the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="192.925" y="656.454">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="213.221" y="656.454">gene [11]. The bio-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="668.472">logical underpinnings of this distinct distribution of Cas4-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="680.434">containing CRISPR-Cas remains obscure.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.686" y="692.453">Phylogenetic analysis indicates that the majority of the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="704.471">CAS-Cas4 proteins form clades that are compatible with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="716.433">the classification of the CRISPR-Cas subtypes delineated</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="728.451">through comparison of effector gene sets. However, there</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.692" y="656.452">are many exceptions to this congruence, so that, in the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="656.452">phylogenetic tree of Cas4, only subtypes I-C, II-B, and V-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="668.47">A are both monophyletic and compact (i.e. do not include</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="680.432">representatives of other subtypes) (Fig. 3, Additional file 1:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="692.451">Figure S1, and Additional file 3: Information File S1). In</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="704.469">agreement with the previous phylogenetic analysis of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="716.431">Cas1, the subtype II-B-associated Cas4 is nested within</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="728.45">the I-B/I-A branch, whereas the subtype V-B-associated</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="740.468">Cas4 proteins, although not forming a clade, fall within</span>
    <span style="fill:#000000;font-family:ddaggl+pttxcbhelvetica-bold;font-size:12.0;font-style:normal;font-weight:bold;" x="128.98" y="114.289">
     <b>a</b>
    </span>
    <span style="fill:#000000;font-family:ddaggl+pttxcbhelvetica-bold;font-size:12.0;font-style:normal;font-weight:bold;" x="128.98" y="328.086">
     <b>b</b>
    </span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.869" y="579.635">Representation of different Cas4 groups in archaea and bacteria.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:8.0;font-style:normal;font-weight:normal;" x="286.242" y="579.635">a</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="290.154" y="579.635">. Presence and absence of CAS-Cas4 and Cas4 solo in completely sequenced</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.703" y="589.613">genomes of archaea and bacteria. Individual genomes were assigned weights inversely proportional to the number of genomes from each species;</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.703" y="599.647">the presence fraction was calculated with genome weights taken into account (see Additional file 4: File S2).</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:8.0;font-style:normal;font-weight:normal;" x="401.555" y="599.647">b</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="406.09" y="599.647">. Numbers of species in the complete</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="609.626">data set containing Cas4 from different CRISPR-Cas4 subtypes and Cas4 solo. The counts were obtained from the complete set of CAS-Cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="619.604">and Cas4-solo (5571 and 702, respectively). Only one strain per species was counted for each respective category of Cas4 assignments to</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="629.638">avoid sequencing bias. See Additional file 7: Table S1 for details. Abbreviation: AM, stand-alone adaptation module</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.141">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.141">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.141">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.061" y="48.141">Page 4 of 14</span>
   </p>
   <div class="figure" title="Fig. 2 ">
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="579.635">Fig. 2</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="104.492">the I-U branch (Additional file 1: Figure S1, Additional file</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="116.453">3: Information File S1) [33] (Fig. 3a, Additional file 1: Fig-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="128.472">ure S1, Additional file 3: Information File S1). Thus, the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="140.49">Cas4-containing adaptation modules in subtypes II-B and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="152.452">V-B apparently were acquired from the respective type I</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="164.471">systems. In contrast, the subtype V-A-associated Cas4 is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="176.489">distinct and does not show clear affinity to any known</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="188.451">CRISPR-Cas systems types (Additional file 1: Figure S1,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="200.47">Additional file 3: Information File S1). This observation is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="212.488">in agreement with the long, deep V-A branch in the re-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.45">spective Cas1 phylogeny [33].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="236.469">Subtypes I-A, I-B, I-D and to a lesser extent I-U, are</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="248.487">scattered among many strongly supported Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="260.449">branches suggesting extensive shuffling of the adaptation</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="272.468">and effector modules (Fig. 3b-f, Additional file 1: Figure S1,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="284.486">Additional file 3: Information File S1). This evidence of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="296.448">module shuffling is consistent with previous observa-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="308.467">tions from phylogenetic analysis of Cas1, although</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="320.485">this effect has been underestimated and was not care-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="332.447">fully explored. With many more genomes now avail-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="344.466">able for analysis, it becomes apparent that adaptation</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="356.484">modules of subtypes I-A, I-B and I-D can function</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="368.446">with (almost) any of the respective effector modules</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="380.465">(Fig. 3, Additional file 1: Figure S1, Additional file 3:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="392.483">Information File S1). Furthermore, these adaptation mod-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="404.445">ules apparently can combine also with type III and type IV</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="416.464">effector modules (Fig. 3, Additional file 1: Figure S1,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="428.482">Additional file 3: Information File S1) because some</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="440.444">branches within well-supported Cas4 clades correspond to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="452.462">these CRISPR-Cas types (Fig. 3, Additional file 1: Figure S1,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="464.481">Additional file 3: Information File S1).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="476.443">We examined in greater detail several cases where</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="488.461">highly similar</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="117.184" y="488.461">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="138.841" y="488.461">genes were associated with different</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="500.48">CRISPR-Cas system (sub)types, especially in closely</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="512.442">related species or strains, where</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="202.167" y="512.442">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="225.127" y="512.442">genes could be</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="524.46">expected to be vertically inherited (Fig. 4). Analysis of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="536.479">these cases suggests that adaptation module genes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="548.441">remain untouched by recombination, whereas effector</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="560.459">genes replace the ancestral effector modules. This infer-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="572.478">ence of the preferred directionality of recombination</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="584.44">events is supported, in particular, by the presence of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="596.458">flanking syntenic regions (Fig. 4). These observations</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="608.42">prompted us to search for similar exchanges within the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="620.439">same CRISPR-Cas subtype. To this end, we clustered the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="632.457">large subunits of the respective systems (Cas8 for type I-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="644.419">A, I-U and I-B and Cas10d for I-D) with permissive cut-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="656.438">offs and mapped these clusters on the tree in order to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="668.456">identify cases where closely related</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="210.784" y="668.456">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="233.007" y="668.456">genes are co-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="680.418">located with substantially different groups of large sub-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="692.437">units of the same subtype. This procedure allowed us to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="704.455">identify additional cases where effector module genes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="716.417">are replaced, whereas the adaptation module remained</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="728.435">unaffected (Fig. 5). In many of these cases, flanking syn-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="104.476">tenic regions allow identification of the precise</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="104.476">boundaries of the effector gene replacement sequences,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="116.438">even on the nucleotide level (Additional file 2: Figure S2).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="128.456">We also noticed some cases when the effector genes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="140.475">are deleted, resulting in emergence of stand-alone</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="152.437">adaptation modules that are usually located inside the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="164.455">branch for the respective subtypes and were identified</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="176.474">in many archaea and bacteria (Fig. 2b, Additional file 1:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="188.435">Figure S1, Additional file 3: Information File S1).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.717" y="200.454">Solo-Cas4 can be clearly differentiated from CAS-Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="212.473">in the phylogenetic tree where they form several distinct</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="224.434">branches that are mostly consistent with the order-level</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="236.453">taxonomy of archaea, suggesting predominant vertical</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="248.472">inheritance (Additional file 1: Figure S1, Additional file</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="260.433">3: Information File S1). Despite the general trend to re-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="272.452">tain solo-Cas4 in the genome, in some archaeal lineages,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="284.471">this gene apparently evolved fast, so that, for example,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="296.432">two groups of</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="370.203" y="296.432">Thermoproteales, Pyrobaculum/Thermo-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="308.451">proteus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="339.363" y="308.451">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="359.489" y="308.451">Vulcanisaeta</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="412.951" y="308.451">, encode dissimilar solo-Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="320.47">sequences that do not cluster with each other or with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="332.431">homologs from other archaea. Several members of the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="344.45">archaeal DPANN superphylum that mostly lack</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="356.469">CRISPR-Cas systems, encode two distinct solo-Cas4 pro-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="368.43">teins (Additional file 1: Figure S1, Additional file 3: In-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="380.449">formation File S1). In bacteria, solo-Cas4 shows a patchy</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="392.467">distribution, with some notable exceptions. First, solo-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="404.429">Cas4 is present and monophyletic in the majority of</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="416.448">Thermoanaerobacteriales</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="411.93" y="416.448">and many</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="456.547" y="416.448">Chloroflexi.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="506.777" y="416.448">Second,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="428.466">solo-Cas4 is encoded in many genomes of the Candidate</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="440.428">Phyla Radiation (CPR), a recently described assemblage</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="452.447">of bacteria with small genomes and many unusual</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="464.465">features, such as</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="379.898" y="464.465">Parcubacteria, Nomurabacteria, Roiz-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="476.427">manbacteria</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="359.943" y="476.427">and others [34, 35]. Some of the solo-Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="488.446">proteins from the CPR confidently group with homologs</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="500.464">from the archaeal DPANN superphylum (Additional file 1:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="512.426">Figure S1, Additional file 3: Information File S1). Thus,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="524.445">solo-Cas4 is clearly over-represented in environmental</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="536.463">bacteria and archaea with small genomes and could per-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="548.425">form important biological functions in these organisms</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="560.444">that remain to be characterized. Furthermore, dissemin-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="572.462">ation of solo-Cas4 genes via HGT between bacteria and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="584.424">archaea appears likely. In several of the DPANN archaeal</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="596.443">genomes, the gene context, namely the adjacency to genes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="608.404">coding for restriction-modification systems, implies that</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="620.423">solo-Cas4 functions in so far uncharacterized defense</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="632.442">pathways (Additional file 7: Table S1).</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="304.724" y="656.454">Cas4 in mobile genetic elements (MGE-Cas4)</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="668.473">Phylogenetic analysis of the Cas4 family identified several</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="680.491">distinct branches that are associated with MGE and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="692.453">viruses, especially, archaeal ones; to our knowledge,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="704.471">the spread of Cas4 homologs in MGE has not been</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="716.49">systematically described previously. In particular, a</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="728.452">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="327.232" y="728.452">gene is present in most of the known viruses of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="740.47">Desulfurococcales and Sulfolobales (Additional file 1:</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.139">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.139">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.139">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.061" y="48.139">Page 5 of 14</span>
   </p>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.133" y="693.021">(See legend on next page.)</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.694" y="48.142">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.862" y="48.142">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.419" y="48.142">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.062" y="48.142">Page 6 of 14</span>
   </p>
   <div class="section" title="a b">
    <span style="fill:#000000;font-family:ddaggl+pttxcbhelvetica-bold;font-size:12.0;font-style:normal;font-weight:bold;" x="76.186" y="115.56">
     <b>a b</b>
    </span>
   </div>
   <div class="section" title="c d">
    <span style="fill:#000000;font-family:ddaggl+pttxcbhelvetica-bold;font-size:12.0;font-style:normal;font-weight:bold;" x="76.186" y="305.189">
     <b>c d</b>
    </span>
   </div>
   <div class="section" title="e f">
    <span style="fill:#000000;font-family:ddaggl+pttxcbhelvetica-bold;font-size:12.0;font-style:normal;font-weight:bold;" x="76.186" y="392.887">
     <b>e f</b>
    </span>
    <div class="figure" title="Fig. 3 ">
     <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="693.021">Fig. 3</span>
    </div>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="212.491">example, Cas4 proteins encoded in Acidianus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.453">filamentous virus genomes form a lineage that is the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="236.472">sister group of solo-Cas4 from</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="193.379" y="236.472">Sulfolobales</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="246.84" y="236.472">within the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="248.49">archaeal solo-Cas4 branch, suggesting acquisition of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="260.452">these genes from the host by the ancestral virus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="272.471">(Additional file 1: Figure S1, Additional file 3: Information</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="284.489">File S1). Sulfolobus spindle shaped virus encodes Cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="296.451">that is most similar to a group of Cas4 proteins associated</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="308.469">with type I CRISPR-Cas systems in</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="227.678" y="308.469">Thermococcales</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="320.488">(Additional file 1: Figure S1, Additional file 3: Information</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="332.45">File S1). In this case, the ancestral virus seems to have</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="344.468">captured a CAS-Cas4 that became solo in the virus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="356.487">genome. Several Cas4 proteins from Sulfolobus islandicus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="368.449">rudiviruses and Acidianus filamentous virus also form a</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="380.467">branch, albeit weakly supported. Cas4 proteins from nu-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="392.486">merous phages form a large, well-supported clade which</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="404.448">includes many phages of Cyanobacteria and Proteo-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="416.466">bacteria, suggesting dissemination of the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="246.84" y="416.466">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="271.388" y="416.466">gene</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="428.485">among phages. One branch in the Cas4 tree cannot</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="440.447">be linked to any known phages but likely corresponds</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="452.465">to uncharacterized prophages integrated in genomes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="464.484">of</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="69.165" y="464.484">Campylobacteriales.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="157.662" y="464.484">Another large clade of Cas4 is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="476.446">apparently associated with conjugative plasmids from</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="488.464">Burkholderiales</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="121.776" y="488.464">. Other Cas4 proteins encoded by</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="500.483">bacteriophages and archaeal viruses are scattered over</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="512.445">the tree and typically do not show clear affinity to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="524.463">solo-Cas4, CAS-Cas4 or each other (Additional file 1:</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="56.693" y="560.473">Horizontal gene transfer and selection pressures on</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="56.693" y="572.492">different groups of</span>
    <span style="fill:#131413;font-family:ddanec+mjfmxwadvtt83913201.i;font-size:9.2;font-style:normal;font-weight:normal;" x="135.495" y="572.492">cas4</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="155.451" y="572.492">genes.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="584.454">To estimate the relative contributions of the vertical and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="596.472">horizontal components in the evolutionary history of the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="608.491">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="83.339" y="608.491">gene family, we compared the tree-induced</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="620.453">distances between Cas4 protein sequences and the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="632.471">distances between the 16S rRNAs from the corresponding</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="644.49">genomes (proxy for the species tree) as described earlier</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="656.452">[11] (see Methods for details); a parallel analysis of Cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="668.47">proteins was used as a reference (see Methods for the link</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="680.489">to the data available online).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="692.45">Both Cas4 and Cas1 only loosely follow the species tree</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="704.469">in their history: the Spearman rank correlation coefficients</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="716.488">with the 16S rRNA distances for CRISPR-Cas associated</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="728.449">Cas4 and Cas1 are 0.15 and 0.22, respectively (see Methods</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="740.468">for details). Notably, both solo</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="186.519" y="740.468">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="207.948" y="740.468">genes and solo</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="273.428" y="200.47">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="200.47">genes show a stronger vertical evolution trend (correlation</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="212.489">coefficients of 0.47 and 0.53, respectively). The substantially</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="224.45">greater horizontal component in the evolutionary history of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="236.469">the CRISPR-associated genes reflects the more dynamic</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="248.488">nature of the evolution of defense systems compared to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="260.449">other categories of microbial genes [36]. The</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="484.098" y="260.449">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="503.997" y="260.449">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="521.458" y="260.449">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="272.468">genes encoded in the same locus nearly always evolve (and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="284.486">get transferred) as a single unit (see Methods for the link to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="296.448">the data available online): the correlation between the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="521.458" y="296.448">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="308.467">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="323.317" y="308.467">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="344.349" y="308.467">distances for co-located genes is 0.88, and both</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="320.485">show closely similar, low correlation coefficients with the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="332.447">rRNA tree (0.14 and 0.13, respectively).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.715" y="344.466">All</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="327.909" y="344.466">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="348.715" y="344.466">genes seem to evolve under comparable levels</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="356.484">of selective pressure, with the medians of the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="512.274" y="356.484">dN/dS</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="368.446">ratios for the CAS-cas4 and solo-cas4 gene pairs be-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="380.465">ing in the range of 0.06 to 0.18 (Additional file 5:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="392.483">Information file S3). These values are close to the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="404.445">genome-wide median</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="404.671" y="404.445">dN/dS</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="438.12" y="404.445">ratios that are in the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="416.464">range of 0.10 to 0.14 [37]. Thus, in agreement with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="428.482">previous observations [38], all</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="442.258" y="428.482">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="465.615" y="428.482">genes experience</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="440.444">moderate levels of purifying selection that are close</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="452.463">to the median values for the respective bacteria and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="464.481">archaea. Furthermore, it has been shown that the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="476.443">characteristic selective pressure on</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="461.59" y="476.443">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="484.778" y="476.443">gene is only</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="488.462">slightly lower than that on</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="429.955" y="488.462">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="453.313" y="488.462">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="473.949" y="488.462">cas2</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="492.034" y="488.462">, the most</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="500.48">strongly conserved</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="394.409" y="500.48">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="414.478" y="500.48">genes, which is compatible</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="512.442">with the joint involvement of the respective Cas pro-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.722" y="524.461">teins in adaptation [38].</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="304.724" y="548.454">Cas4-like proteins of pfam12705</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="560.473">As mentioned above, Cas4-like proteins that showed the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="572.491">highest similarity to pfam12705 are not associated with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="584.453">CRISPR-Cas systems. However, in addition to the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="596.471">Campylobacter</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="370.204" y="596.471">phages that encode a Cas4-like nuclease</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="608.49">that has been shown to interfere with the host type II-C</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="620.452">CRISPR-Cas system [22, 23], members of this family</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="632.47">were identified in several other notable contexts. These in-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="644.489">clude a large clade that consists of phages infecting differ-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="656.451">ent</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="321.052" y="656.451">Mycobacteria</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="378.255" y="656.451">as well as some</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="445.889" y="656.451">Propionibacterium</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="523.728" y="656.451">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="668.469">Bacillus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="339.193" y="668.469">phages as well as several smaller branches includ-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="680.488">ing phages that infect</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="399.683" y="680.488">Bacillus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="431.545" y="680.488">,</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="437.1" y="680.488">Thermus</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="476.842" y="680.488">and other bac-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="692.45">teria (Additional file 1: Figure S1, Additional file 3:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="704.468">Information File S1).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.717" y="716.487">The largest branch in the pfam12705 family includes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="728.449">proteins from many bacteria and mesophilic archaea, in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="740.467">which a Cas4-like nuclease domain is fused to a UvrD-</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="109.367">(See figure on previous page.)</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.869" y="119.345">Selected examples of confidently supported subtrees of Cas4 associated with different subtypes of CRISPR-Cas effector complexes. Strongly</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="129.38">supported (support values indicated in red) subtrees correspond to branches from the complete Cas4 tree shown in Additional file 1: Figure S1</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="139.358">and available in Newick format as Supplementary Information File 1.Each sequence is denoted by a numeric identifier, matching those in</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="149.392">Additional file 7: Table S1, subtype of the CRISPR-Cas system from the respective locus and species name. Several branches were collapsed and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="159.37">are indicated by triangles with CRISPR-Cas system subtype indicated next to respective triangle. The coloring of the sequences corresponds to</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="169.347">distinct CRISPR-Cas system types or subtypes according to the color code provided at the bottom</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.138">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.138">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.138">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.061" y="48.138">Page 7 of 14</span>
   </p>
   <div class="figure" title="Figure S1, Additional file 3: Information File S1). For">
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="200.473">Figure S1, Additional file 3: Information File S1). For</span>
    <div class="figure" title="Figure S1, Additional file 3: Information File S1).">
     <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="536.482">Figure S1, Additional file 3: Information File S1).</span>
     <div class="figure" title="Fig. 3 ">
      <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="119.345">Fig. 3</span>
     </div>
    </div>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.132" y="711.162">(See legend on next page.)</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.694" y="48.142">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.862" y="48.142">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.419" y="48.142">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.062" y="48.142">Page 8 of 14</span>
   </p>
   <div class="figure" title="Fig. 4 ">
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="711.162">Fig. 4</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="200.473">like superfamily 1 (SF1) 3</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="162.879" y="200.473">′</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="175.125" y="200.473">′</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="182.211" y="200.473">helicase (Fig. 6). Fusion of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="212.491">RecB family nucleases with SF1 helicases is common,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.453">e.g. in RecB and AddB, key proteins in bacterial</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="236.472">recombinational repair pathways [39]. The UvrD</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="248.49">helicase has been originally identified as a component of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="260.452">the nucleotide excision repair complex along with UvrA</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="272.471">and UvrB proteins, but subsequently, has been shown to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="284.489">participate in the regulation of RecA recombinase,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="296.451">mismatch repair, transcription-coupled repair, and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="308.469">chromosome and plasmid replication, as an accessory</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="320.488">helicase [39, 40]. Many bacteria encode multiple UvrD</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="332.45">(COG0210) paralogs [41].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="344.468">We constructed a tree of UvrD homologs from all ge-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="356.487">nomes that encode the UvrD-Cas4-like fusion and found</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="368.449">that the helicase domains of the fusion proteins are also</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="380.467">monophyletic, indicating a single origin and, most likely,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="392.486">a unique function of the fusion proteins (Additional file 6:</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="404.448">Information File S4). The UvrD-Cas4 proteins are</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="416.466">encoded in two distinct, conserved contexts. The most</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="428.485">common genomic neighborhood includes genes for</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="440.447">methylation (Mod) and restriction (Res) subunits of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="452.465">Type III Restriction-Modification systems [42</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="269.461" y="452.465">–</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="275.413" y="452.465">44].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="464.484">Several of these loci also encompass genes for predicted</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="476.446">regulatory proteins containing WYL and helix-turn-helix</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="488.464">(HTH) domains as well as Shlaffen-like ATPases fused</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="500.483">to an HTH domain that are often encoded in other</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="512.445">defense contexts.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="524.463">Another context of the UvrD-Cas4 fusion includes</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="536.482">DNA-cytosine methylase Dcm that is responsible for</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="548.444">all 5-methylcytosine modifications in</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="234.254" y="548.444">E. coli</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="271.502" y="548.444">[45].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="560.462">The physiological role of this methylation remains</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="572.481">unknown although it has been shown that Dcm</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="584.442">recognition sites correspond to sequence motifs for</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="596.461">very short patch repair of T/G mismatches, and the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="608.423">Vsr nucleases involved in this process are often</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="620.441">encoded next to the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="153.127" y="620.441">dcm</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="176.031" y="620.441">gene [45]. Indeed, several</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="632.46">uvrD-cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="109.247" y="632.46">loci also contain a gene for a Vsr</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="644.422">nuclease, suggesting that this system retains the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="656.44">same features of recognition sites and functional link</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="668.459">between Dcm and Vsr (Fig. 6). The same pair of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="680.421">genes was observed previously in a variety of defense</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="692.439">gene contexts, MORC family ATPases in particular</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="704.458">[46]. Collectively, these genomic associations strongly</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="716.42">suggest that the UvrD-Cas4 helicase-nuclease fusion</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="728.438">proteins are components of multiple microbial</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="200.459">defense systems.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.717" y="200.459">Considering that, in both cases described above, the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="212.477">UvrD-Cas4 protein appears to belong to the same</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="224.439">subfamily and the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="393.277" y="224.439">uvrD-cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="442.6" y="224.439">gene is either stand-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="236.458">alone or is found in non-conserved contexts in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="248.476">archaeal and bacterial genomes, it appears that the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="260.438">fusion protein plays an ancillary role in different</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="272.457">pathways by facilitating site-specific DNA unwinding</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="284.475">and cleavage. The Cas4-like nuclease domain can be</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="296.437">predicted to play a role analogous to the nuclease</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="308.456">domains of RecB and AddB in promoting DNA</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="320.474">recombination pathways [19]. Most of the bacterial</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="332.436">genomes that encode the UvrD-Cas4 fusion also</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="344.455">encode RecB (a near ubiquitous protein in bacteria)</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="356.473">suggesting that the functions of these helicase-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="368.435">nuclease fusion proteins could be partially redundant</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="380.454">rather than complementary, a pattern typical of repair</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="392.472">pathways [47, 48].</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.717" y="404.434">The link between UvrD and Cas4-like protein evolved</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="416.453">at least on one other independent occasion. In several</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="428.471">bacteria, mostly Clostridiales, the two genes are not</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="440.433">fused but are encoded in a predicted operon in a context</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="452.452">that does not appear indicative of defense functions, sug-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="464.47">gesting that the two enzymes could be jointly involved</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="476.432">in other cellular processes, such as DNA repair (Fig. 6).</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.717" y="488.451">Finally, in several bacteria and mesophilic archaea, we</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="500.469">identified another link between a Cas4-like nuclease and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="512.431">a superfamily 2 helicase that belongs to the uncharacter-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="524.449">ized YprA-like family (COG1205) and is unrelated to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="536.468">UvrD. This helicase contains a characteristic C-terminal</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="548.43">metal-binding domain (pfam09369/DUF1998 family)</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="560.448">and large inserts with additional putative metal-binding</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="572.467">domains (Fig. 6). Helicases of this family are strongly</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="584.429">associated with defense islands [49], suggesting that, to-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="596.447">gether with the Cas4-like protein, they participate in so</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="608.409">far uncharacterized defense pathways.</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="304.724" y="632.473">Conclusions</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="644.491">Unlike most</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="361.36" y="644.491">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="378.821" y="644.491">genes, Cas4 family endonucleases are</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="656.453">about as often encoded outside as they are within</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="668.472">CRISPR-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="340.611" y="668.472">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="359.489" y="668.472">loci and are also found in many MGE,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="680.49">particularly archaeal viruses. Because of this unusual</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="692.452">distribution of</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="373.663" y="692.452">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="398.041" y="692.452">genes in bacterial and archaeal</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="704.471">genomes, we sought to explore the phylogenomics of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="716.489">the Cas4 family. The results of this phylogenomic indi-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="728.451">cate that evolution of those</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="428.201" y="728.451">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="450.481" y="728.451">genes that belong to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="740.47">CRISPR-Cas adaptation modules (Cas-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="477.41" y="740.47">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="495.098" y="740.47">) involves</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="109.367">(See figure on previous page.)</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.869" y="119.345">Selected examples of in situ replacement of effector complex genes in CRISPR-Cas systems of different subtypes. For each locus, species</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="129.38">name, genome accession number and the respective nucleotide coordinates are indicated. The genes of a representative locus are shown by</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="139.358">arrows. The arrow indicates the transcription direction of the respective gene. Genes and CRISPR arrays are shown roughly to scale. Homologous</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="149.392">genes are connected by dotted lines and amino acid sequence identity (%) is indicated. Asterisks indicate cases where the overall sequence</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="159.37">identity was very low, and the percent identity was taken from a small aligned fragment. Cas4 genes are shown in green. The region of likely in</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="169.347">situ replacement is shown by a blue rectangle</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.138">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.138">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.138">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="496.061" y="48.138">Page 9 of 14</span>
   </p>
   <div class="figure" title="Fig. 4 ">
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="119.345">Fig. 4</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.869" y="711.162">Selected examples of in situ replacement of effector complex genes in CRISPR-Cas loci of the same subtype. Designations are the same as</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="721.14">in the Fig. 4</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.142">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.142">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.142">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="491.979" y="48.142">Page 10 of 14</span>
   </p>
   <div class="figure" title="Fig. 5 ">
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="711.162">Fig. 5</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="121.638" y="549.6">UvrD Cas4-like PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="210.53" y="447.858">ResMod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="219.767" y="447.858">UvrD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="135.231" y="543.304">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="188.415" y="543.304">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="101.517" y="523.707">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="106.565" y="503.146">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="101.517" y="481.992">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="115.634" y="523.708">UvrD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="140.007" y="543.304">HTH</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="232.78" y="543.304">SbcD SbcC YwqK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="333.007" y="548.871">antitoxin</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="327.397" y="475.351">Ferroplasma_acidarmanus_fer1_GCA_000152265.2 CP004145.1 24305..21531</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="325.869" y="515.143">Anaerococcus_hydrogenalis_DSM_7454_GCA_000173355.1 ABXA01000031.1 90693..85891</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="77.056" y="536.341">GNAT RsuA WcaJ</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="357.709" y="536.341">Fusobacterium_nucleatum_vincentii_ChDC_F8_GCA_001296125.1 CP012714.1 1238169..1251591</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="95.097" y="437.805">AlaX RlmN MrcB AANH</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="325.869" y="437.805">Heliobacterium_N15_MGS_306_GCA_000980455.1 LARK01000032.1 120119..111687</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="154.329" y="557.275">Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="293.646" y="557.275">Methanobrevibacter_smithii_ATCC_35061_GCA_000016525.1 CP000678.1 58788..54271</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="325.989" y="496.366">Streptomyces_CdTB01_CdTB01_GCA_001484565.1 CP013743.1 6229954..6232920</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="327.397" y="455.418">Brachyspira_CAG_484_GCA_000431315.1 HF989736.1 220325..215813</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="139.563" y="419.676">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="126.177" y="400.122">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="103.641" y="380.609">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="128.209" y="342.345">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="293.427" y="332.724">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="369.946" y="332.724">Dehalococcoides_mccartyi_BTF08_GCA_000341695.1 CP004080.1 279971..265714</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="113.122" y="342.346">Methyltransferase PglZ Lon Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="88.486" y="380.194">Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="153.332" y="380.194">Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="98.78" y="419.674">Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="90.846" y="400.023">Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="85.153" y="342.345">Vsr</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="334.638" y="342.345">Vsr PD-D/ExK</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="180.646" y="354.437">Unknown</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="319.466" y="354.437">Prosthecochloris_aestuarii_DSM_271_GCA_000020625.1 CP001108.1 2492066..2488075</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="326.679" y="372.527">Ralstonia_pickettii_12D_GCA_000023425.1 CP001644.1 216728..220915</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="328.367" y="412.329">Sulfobacillus_acidophilus_DSM_10332_GCA_000237975.1 CP003179.1 62252..67746</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="328.698" y="391.783">Burkholderia_ubonensis_MSMB22_GCA_000959245.1 CP009488.1 2411930..2407973</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="129.842" y="126.217">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="273.295" y="126.217">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="106.704" y="169.161">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="107.474" y="189.008">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="103.999" y="208.773">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="106.705" y="228.361">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="103.999" y="248.469">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="108.698" y="272.114">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="145.703" y="126.166">UvrD Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="239.942" y="126.166">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="238.971" y="149.505">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="213.238" y="168.005">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="222.578" y="208.053">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="247.478" y="208.053">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="209.232" y="232.359">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="213.392" y="271.516">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="250.607" y="271.516">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="257.117" y="117.186">PD-D/ExK</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="366.005" y="117.186">Halanaerobium_hydrogeniformans_GCA_000166415.1 CP002304.1 1170463..1184174</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="112.18" y="126.165">COG3472</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="216.924" y="126.165">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="217.119" y="148.669">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="218.268" y="270.181">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="229.938" y="270.181">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="235.781" y="290.026">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="204.853" y="187.826">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="194.553" y="228.128">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="250.607" y="228.128">FRG</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="250.607" y="233.696">domain</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="193.221" y="248.469">Res</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="158.738" y="148.669">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="183.136" y="148.669">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="156.912" y="168.004">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="158.974" y="187.826">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="164.678" y="208.053">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="154.0" y="228.361">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="154.977" y="270.122">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="188.337" y="290.699">Mod</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="199.212" y="127.58">Mod</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="324.547" y="127.58">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="147.501" y="139.114">Unknown</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="329.421" y="139.114">Herminiimonas_arsenicoxydans_GCA_000026125.1 CU207211.1 2204374..2194364</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="88.336" y="150.99">HTH-WYL</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="84.428" y="159.156">HTH-WYL</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="329.421" y="159.156">Anoxybacillus_flavithermus_WK1_GCA_000019045.1 CP000922.1 1159497..1166737</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="328.069" y="177.468">Nitrosomonas_eutropha_C91_GCA_000014765.1 CP000452.1 22010..14424</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="326.902" y="200.398">Aciduliprofundum_MAR08_339_MAR08_339_GCA_000327505.1 CP003168.1 1095779..1086773</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="327.762" y="220.239">Roseburia_hominis_A2_183_GCA_000225345.1 CP003040.1 2833946..2823521</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="225.528" y="228.361">pfam11185 Schlafen-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="268.416" y="240.346">ATPase+HTH</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="327.763" y="240.346">Selenomonas_oral_taxon_136_F0591_GCA_001554015.1 CP014239.1 43923..51316</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="327.763" y="263.676">Spirosoma_linguale_DSM_74_GCA_000024525.1 CP001770.1 46355..37201</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="327.035" y="283.258">Vibrio_nigripulchritudo_SFn1_GCA_000801275.1 FO203527.1 2008548..1999051</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="319.822" y="303.756">Caldimicrobium_thiodismutans_TF1_GCA_001548275.1 AP014945.1 384910..376763</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="267.895" y="642.97">Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="82.505" y="642.972">Dcm</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="285.505" y="642.972">Vsr</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="260.148" y="622.369">Vsr</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="99.073" y="597.683">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="85.997" y="620.846">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="87.656" y="646.321">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="106.704" y="665.755">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="101.656" y="665.755">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="141.437" y="665.755">Unknown</span>
    <span style="fill:#db291a;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="135.369" y="599.621">Unknown</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="269.67" y="599.621">Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="257.795" y="595.947">DUF1998MBD MBD MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="160.268" y="619.163">MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="154.977" y="661.751">MBD MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="189.768" y="661.751">MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="180.154" y="661.751">MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="199.596" y="661.751">MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="188.553" y="684.179">MBD MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="247.477" y="661.751">MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="169.898" y="619.163">MBD MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="216.262" y="622.784">MBD</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="236.784" y="664.074">Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="266.641" y="664.074">Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="236.926" y="590.16">Cas4-like</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="319.822" y="590.16">Mesorhizobium_F7_F7_GCA_000798645.1 JTGD01000043.1 185152..174401</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="163.454" y="595.947">COG1205</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="163.454" y="601.516">helicase</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="132.483" y="620.846">COG1205</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="132.483" y="664.073">helicase</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="163.454" y="664.073">COG1205</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="163.454" y="669.641">helicase</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="124.564" y="641.988">COG1205</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="124.564" y="687.529">helicase</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="261.927" y="687.53">Cas4-likeCOG1205</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:4.64;font-style:normal;font-weight:normal;" x="140.146" y="613.653">helicase</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="305.657" y="613.653">Methanosarcina_soligelidi_SMA_21_GCA_000744315.1 JQLR01000001.1 3755367..3765130</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="310.488" y="657.016">Mycobacterium_novocastrense_JCM18114_GCA_001570485.1 BCTA01000023.1 125399..114759</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="309.413" y="634.69">Acidobacteriaceae_bacterium_KBS_96_KBS_96_GCA_000381625.1 KB906767.1 2108185..2119739</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:3.87;font-style:normal;font-weight:normal;" x="305.659" y="678.866">Deinococcus_swuensis_DY59_GCA_000800395.1 CP010028.1 1852609..1841920</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="225.818" y="619.163">DUF1998</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="223.524" y="662.525">DUF1998</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="255.502" y="662.525">DUF1998</span>
    <span style="fill:#070705;font-family:njytxmhelvetica;font-size:2.32;font-style:normal;font-weight:normal;" x="250.64" y="684.18">DUF1998</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.132" y="711.502">(See legend on next page.)</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.694" y="48.142">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.862" y="48.142">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.419" y="48.142">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="491.98" y="48.142">Page 11 of 14</span>
   </p>
   <div class="figure" title="Fig. 6 ">
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="711.502">Fig. 6</span>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.454">extensive horizontal transfer that, however, almost al-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="236.472">ways involves complete adaptation modules. The evolu-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="248.491">tionary integrity of the adaption modules sharply</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="260.453">contrasts the rampant modular shuffling of CRISPR-Cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="272.471">systems whereby a given variant of the adaptation mod-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="284.49">ule can combine with virtually any variety of effector</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="296.452">modules. The evolutionary history of solo-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="236.011" y="296.452">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="257.611" y="296.452">genes is</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="308.47">much better compatible with the microbial species tree</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="320.489">and appears to include fewer horizontal gene transfer</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="332.451">events. The Cas-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="127.559" y="332.451">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="149.273" y="332.451">and solo-</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="188.448" y="332.451">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="210.161" y="332.451">genes evolve under</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="344.469">moderately strong purifying selection that, for both</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="356.488">classes of</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="101.028" y="356.488">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="123.251" y="356.488">genes, is close to the genomic median.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="368.449">Thus, the evolutionary regime of the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="217.248" y="368.449">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="239.131" y="368.449">genes, along</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.694" y="380.468">with</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="77.726" y="380.468">cas1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="98.533" y="380.468">and</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="116.674" y="380.468">cas2</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="137.48" y="380.468">genes, resembles that of</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="240.265" y="380.468">‘</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="242.249" y="380.468">regular</span>
    <span style="fill:#131413;font-family:ddaeee+ryrlpvadvtt86d47313+20;font-size:9.8;font-style:normal;font-weight:normal;" x="271.163" y="380.468">’</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="276.209" y="380.468">mi-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="392.487">crobial genes involved in various cellular functions, with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="404.448">no evidence of direct involvement in virus-host arms</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="416.467">races. However,</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="128.241" y="416.467">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="151.315" y="416.467">genes were recruited by various</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="428.486">MGE on many independent occasions, suggesting that</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="440.447">Cas4 nucleases might be involved in anti-defense func-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="452.466">tions. The solo-Cas4 as well as MGE-encoded Cas4 so</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="464.485">far have not been studied experimentally. Functional</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="476.446">characterization of these proteins is likely to result in</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="488.465">identification of novel defense and anti-defense systems,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.695" y="500.484">and their connections with CRISPR-Cas systems.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="56.693" y="536.491">Data</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="548.454">The prokaryotic genomic dataset was taken from</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="560.472">WGS and GenBank repositories at the NCBI as of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="572.491">March 2016 and includes 43,000 genomes. Cas4-like</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="584.453">proteins were identified using PSI-BLAST search</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="596.471">against sequence profiles corresponding to the follow-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="608.49">ing subfamilies: COG1468, COG4343, pfam01930,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="620.452">pfam06023, cd09637, cd09659, cls000170 [11] and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="632.47">pfam12705. To ensure specificity of the extracted set</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="644.489">of proteins after the initial identification, all retrieved</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="656.45">protein sequences were additionally run against the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="668.469">Conserved Domain Database (CDD) using the RPS-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="680.488">BLAST program with the same parameters and all</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="692.449">proteins with higher similarity to any other profiles</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="704.468">were discarded. The protein sequences from the</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="272.579" y="704.468">cas4</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="716.487">neighborhoods were extracted from the respective ge-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="728.448">nomes or contigs and annotated using the COGs,</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.45">Pfam and CD profiles from the CDD and the custom</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="224.45">defense gene profiles [11]. The</span>
    <span style="fill:#131413;font-family:ddaeef+xqlbfpadvtt8861b38f.i;font-size:9.8;font-style:normal;font-weight:normal;" x="468.282" y="224.45">cas</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="490.789" y="224.45">genes and</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="236.469">CRISPR-Cas subtypes were annotated according to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.723" y="248.487">the previously described nomenclature [11].</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:9.2;font-style:normal;font-weight:normal;" x="304.724" y="272.473">Clustering and Phylogenetic analysis</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="284.492">The NCBI BLASTCLUST program (ftp://ftp.ncbi.nih.-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="296.453">gov/blast/documents/blastclust.html) was used to cluster</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="308.472">sequences by similarity. For Cas4 clustering, BLAS-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="320.49">TCLUST was used with the sequence identity threshold</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="332.452">of 50% and length coverage threshold of 0.8; for 16S</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="344.471">rRNA nucleotide sequences, the program was used with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="356.489">sequence identity threshold of 90% and length coverage</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="368.451">threshold of 0.9. For Cas8 and Cas10d clustering, the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="380.47">program was used with sequence identity threshold of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="392.488">0.5 and length coverage threshold of 80%. Alignments of</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="404.45">protein sequences were constructed using MUSCLE [50]</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="416.469">and MAFFT [51] programs. Short fragments or dis-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="428.487">rupted sequences were discarded. Sites with the gap</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="440.449">character fraction values &gt;0.5 and homogeneity &lt;0.1</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="452.468">were removed from the alignment. Phylogenetic analysis</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="464.486">was performed using the FastTree program [52], with</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="476.448">the WAG evolutionary model for amino acid sequences</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="488.467">and the GTR evolutionary model for nucleotide</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="500.485">sequences. The same program was used to compute</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="512.447">bootstrap values.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.718" y="524.466">Relationships within diverse sequence families were</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="536.484">established using the following procedure: initial sequence</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="548.446">clusters were obtained using UCLUST [53] with the se-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="560.465">quence similarity threshold of 0.5; sequences were aligned</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="572.483">within clusters using MUSCLE [50]. Then cluster-to-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="584.445">cluster similarity scores were obtained using HHsearch</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="596.464">[54] (including trivial clusters consisting of a single se-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="608.482">quence each); a UPGMA dendrogram was constructed</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="620.444">from the pairwise similarity scores. Highly similar clusters</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="632.462">(pairwise score to self-score ratio &gt; 0.1) were aligned to</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="644.481">each other using HHALIGN [55]; the procedure was re-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="656.443">peated iteratively. At the last step, sequence-based trees</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="668.461">were reconstructed from the cluster alignments using</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="680.48">FastTree [52], as described above and rooted by mid-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="692.442">point; these trees were grafted onto the tips of the profile</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="704.46">similarity-based UPGMA dendrogram.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="312.718" y="716.479">For the comparison of evolutionary distances between</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="728.441">Cas4 and Cas1 protein sequences and 16S rRNA se-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="304.724" y="740.459">quences from the corresponding genomes (proxy of the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="109.367">(See figure on previous page.)</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="84.869" y="119.345">Organization of loci containing UvrD-like, COG1205 and Cas4-like genes. For each locus, species name, genome accession number and the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="129.38">respective nucleotide coordinates are indicated. The genes of a representative locus are shown by arrows. The scale of an arrow is roughly</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="139.358">proportional to gene length. The arrow indicates direction of the respective gene. Homologous genes and domains are color-coded. Four panels</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="149.392">separated by dotted rectangles show a common theme of locus organization. Abbreviations and gene names: Mod</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:8.0;font-style:normal;font-weight:normal;" x="437.214" y="149.392">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="441.182" y="149.392">modification subunit of type</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="159.37">III RM system, Res</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:8.0;font-style:normal;font-weight:normal;" x="121.493" y="159.37">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="127.842" y="159.37">restriction enzyme of type III RM system; Dcm - DNA-cytosine methylase; HTH- helix turn helix; Vsr</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:8.0;font-style:normal;font-weight:normal;" x="445.378" y="159.37">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="451.727" y="159.37">very short patch repair</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="169.348">nuclease, PD-DExK</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:8.0;font-style:normal;font-weight:normal;" x="124.044" y="169.348">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="130.45" y="169.348">PD-DExK family nuclease; WcaJ-Sugar transferase involved in LPS biosynthesis; RsuA - Pseudouridine synthase; MrcB -</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="179.382">Membrane carboxypeptidase; RlmN - Adenine C2-methylase of 23S rRNA A2503 and tRNA A37; AlaX - Ser-tRNA(Ala) deacylase; AANH - Adenine</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="189.361">nucleotide alpha hydrolases</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.139">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.139">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.139">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="491.979" y="48.139">Page 12 of 14</span>
   </p>
   <div class="section" title="Methods">
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="56.693" y="524.473">Methods</span>
    <div class="figure" title="Fig. 6 ">
     <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:8.0;font-style:normal;font-weight:normal;" x="62.702" y="119.345">Fig. 6</span>
    </div>
   </div>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="104.492">species tree), all pairwise distances between CRISPR-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="116.453">associated or solo proteins were calculated from the cor-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="128.472">responding phylogenetic trees; for protein sequences</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="140.49">from unaligned clusters, an arbitrary high distance of 12</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="152.452">was assigned to the corresponding pair. If a pair of ge-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="164.471">nomes contained more than one Cas4 or Cas1 sequence</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="176.489">in either or both of them, the shortest protein-protein</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="188.451">distance was selected to represent this genome pair.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="200.47">Distances between the corresponding 16S rRNA se-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="212.488">quences were, likewise, calculated from the 16S rRNA</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="224.45">trees or, if the two 16S rRNAs belonged to unaligned</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="236.469">clusters, an arbitrary distance high distance of 3 was</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="248.487">assigned. Spearman rank correlation between protein-</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="260.449">protein and rRNA-rRNA sequence distances for all pairs</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="272.468">was reported as a measure of coherence between the</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="284.486">protein evolution and the species tree.</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="64.687" y="296.448">All pairwise distances are available for download at</span>
    <span style="fill:#131413;font-family:ddaede+nwnvggadvtt86d47313;font-size:9.8;font-style:normal;font-weight:normal;" x="56.693" y="308.467">ftp://ftp.ncbi.nih.gov/pub/wolf/_suppl/cas4.</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="56.693" y="334.438">Additional files</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="356.718">Additional file 1: Figure S1.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="156.246" y="356.718">Schematic representation of the maximum</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="365.732">likelihood phylogenetic tree of Cas4 (7060 sequences all together), available</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="374.746">in the Supplementary File 1. Support values are calculated by FastTRee</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="383.704">program only for the confidently aligned groups, all other values were</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="392.718">assigned to zero automatically. Major well-supported distinct branches are</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="401.732">shown by rectangles which are color-coded according to Cas4 assignments.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="410.746">Assignments and other comments are shown next to the each collapsed</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="419.703">branch. Individual sequences in the tree are described by a local numeric ID,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="428.717">species name and color-coded according to Cas4 assignment (also provided</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="437.731">in the Additional file 7: Table S1). Blue shading shows tree clades</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="446.746">that belong to pfam12705 family. (PDF 694 kb)</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="458.708">Additional file 2: Figure S2.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="156.246" y="458.708">Nucleotide sequence comparisons of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="467.722">CRISPR-Cas loci encoded in closely related strains. On the axes, the labels</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="476.736">contain the name of the source genome, contig ID and the coordinates of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="485.693">the respective loci. The annotations for CRISPR-Cas loci were taken from the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="494.707">Additional file 7: Table S1,</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="138.557" y="494.707">“</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="140.655" y="494.707">Loci</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="152.504" y="494.707">”</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="156.643" y="494.707">worksheet. The cartoons on the axes repre-</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="503.721">sent the genes and CRISPR repeats encoded in these loci. The sizes of the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="512.736">cartoons are proportional to the actual sizes of these genes. Colors: black</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="521.693">are CRISPR arrays, blue are cas genes, green - cas4 gene, shaded area are</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="530.707">the regions which have &gt;70% sequence identity level. Left: Comparison of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="539.721">two I-C systems from Marinobacter strains. Right: Comparison of two I-B</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="548.735">systems from Campylobacter strains. (DOCX 606 kb)</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="560.697">Additional file 3: Information File S1.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="188.9" y="560.697">Complete tree for Cas4-like set</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="569.712">in Newick format. Cas4 assignments are included in the leaf name.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="578.726">(TXT 463 kb)</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="590.744">Additional file 4: Information File S2.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="188.9" y="590.744">Breakdown of CAS-Cas4 and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="599.702">solo-Cas4 presence in completely sequenced genomes. (XLSX 248 kb)</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="611.721">Additional file 5: Information File S3.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="188.9" y="611.721">Detailed description of the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.965" y="620.735">Cas4</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:7.5;font-style:normal;font-weight:normal;" x="78.123" y="620.735">dN/dS</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="98.136" y="620.735">analysis, table with results. (DOCX 51 kb)</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="632.697">Additional file 6: Information File S4.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="188.901" y="632.697">Complete tree for selected</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="641.711">representatives of UvrD family in Newick format. The tree is based on the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="650.726">helicase domain alignment. Protein linked to Cas4 are indicated in the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="659.74">respective leaf names. (DOCX 51 kb)</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="671.702">Additional file 7: Table S1</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="151.371" y="671.702">Worksheet</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="185.216" y="671.702">“</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="187.427" y="671.702">loci</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="197.802" y="671.702">”</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="202.281" y="671.702">provides detailed</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="680.716">information on all cas4 loci in completely sequenced and draft genomes</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="689.73">of archaea and bacteria. Annotation for the proteins encoded in the loci</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="698.744">is based on Cas protein and CDD assignments using PSI-BLAST program</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="707.702">(see Methods for details). Worksheet</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="173.595" y="707.702">“</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="175.806" y="707.702">tree order and assignments</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="258.804" y="707.702">”</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="716.716">provides information of the order of the Cas4 in the tree (Supplementary file 1),</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="725.73">Cas4 assignments to distinct groups of CAS-Cas4, solo-Cas4 and others.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="61.966" y="104.489">(TXT 28 kb)</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="104.489">Acknowledgements</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="113.446">The authors thank Koonin group members for helpful discussions.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="131.474">Funding</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="140.489">This work was supported by intramural funds of the US Department of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="149.446">Health and Human Services (the National Library of Medicine, to EVK) and by</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="158.46">the NIH grant R35 118,160 (to MPT).</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="176.488">Availability of data and materials</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="185.445">The datasets supporting the conclusions of this article are included within the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="194.459">additional files and available at ftp://ftp.ncbi.nih.gov/pub/wolf/_suppl/cas4</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.725" y="212.488">Authors</span>
    <span style="fill:#131413;font-family:ddbfkb+jcrgwhadvtt99c4c969+20;font-size:7.5;font-style:normal;font-weight:normal;" x="330.52" y="212.488">’</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="334.488" y="212.488">contributions</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="221.445">Conception and design of the study: KSM, YIW, EVK. Data collection: SH, SS.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="230.46">Data analysis: SH, YIW, MPT, KSM. Manuscript drafting: SH, YIW, KSM.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="239.474">Manuscript revision for critical intellectual content: EVK. All authors read and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="248.488">approved the final manuscript.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="266.46">Ethics approval and consent to participate</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="275.474">Not applicable.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="293.446">Consent for publication</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="302.46">Not applicable.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="320.488">Competing interests</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="329.446">The authors declare that they have no competing interests.</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="304.724" y="353.487">Publisher</span>
    <span style="fill:#131413;font-family:ddbfkc+rppbkgadvttaf7f9f4f.b+20;font-size:10.3;font-style:normal;font-weight:normal;" x="347.925" y="353.487">’</span>
    <span style="fill:#131413;font-family:ddadlb+lqwndqadvttaf7f9f4f.b;font-size:10.3;font-style:normal;font-weight:normal;" x="350.476" y="353.487">s Note</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="362.444">Springer Nature remains neutral with regard to jurisdictional claims in</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="371.458">published maps and institutional affiliations.</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="389.487">Author details</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="307.389" y="398.444">National Center for Biotechnology Information, National Institutes of Health,</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="404.113">Bethesda, MD, USA.</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="368.447" y="407.458">Skolkovo Institute of Science and Technology, Skolkovo</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="413.128">143025, Russia.</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="354.047" y="416.473">Departments of Biochemistry and Molecular Biology,</span>
    <span style="fill:#000000;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="425.487">Genetics, and Microbiology, University of Georgia, Athens, GA, USA.</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="443.459">Received: 7 August 2017 Accepted: 16 November 2017</span>
    <span style="fill:#131413;font-family:ddaecb+xvvsfkadvtt99c4c969;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="470.444">References</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="479.458">1. Plagens A, Tjaden B, Hagemann A, Randau L, Hensel R. Characterization of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="488.472">the CRISPR/Cas subtype I-A system of the hyperthermophilic crenarchaeon</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="497.487">Thermoproteus tenax. J Bacteriol. 2012;194(10):2491</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="467.093" y="497.487">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="470.665" y="497.487">500.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="506.444">2. Lemak S, Beloglazova N, Nocek B, Skarina T, Flick R, Brown G, Popovic A,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="515.459">Joachimiak A, Savchenko A, Yakunin AF. Toroidal structure and DNA cleavage</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="524.473">by the CRISPR-associated [4Fe-4S] cluster containing Cas4 nuclease SSO0001</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="533.487">from Sulfolobus solfataricus. J Am Chem Soc. 2013;135(46):17476</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="498.161" y="533.487">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="501.676" y="533.487">87.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="542.444">3. Li M, Wang R, Zhao D, Xiang H. Adaptation of the Haloarcula hispanica</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="551.458">CRISPR-Cas system to a purified virus strictly requires a priming process.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="560.472">Nucleic Acids Res. 2014;42(4):2483</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="416.58" y="560.472">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="420.151" y="560.472">92.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="569.486">4. Liu T, Li Y, Wang X, Ye Q, Li H, Liang Y, She Q, Peng N. Transcriptional</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="578.443">regulator-mediated activation of adaptation genes triggers CRISPR de novo</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="587.458">spacer acquisition. Nucleic Acids Res. 2015;43(2):1044</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="470.948" y="587.458">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="474.463" y="587.458">55.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="596.472">5. Amitai G, Sorek R. CRISPR-Cas adaptation: insights into the mechanism of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="605.486">action. Nat Rev Microbiol. 2016;14(2):67</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="431.546" y="605.486">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="435.118" y="605.486">76.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="614.443">6. Jackson SA, McKenzie RE, Fagerlund RD, Kieper SN, Fineran PC, Brouns SJ:</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="623.457">CRISPR-Cas: Adapting to change</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="416.182" y="623.457">. Science 2017,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="471.117" y="623.457">(6333).</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.723" y="632.471">7. Barrangou R, Horvath P. A decade of discovery: CRISPR functions and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="641.485">applications. Nat Microbiol. 2017;2:17092.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.723" y="650.443">8. Barrangou R, Marraffini LA. CRISPR-Cas systems: prokaryotes upgrade to</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="659.457">adaptive immunity. Mol Cell. 2014;54(2):234</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="443.621" y="659.457">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="447.192" y="659.457">44.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.723" y="668.471">9. Marraffini LA. CRISPR-Cas immunity in prokaryotes. Nature. 2015;</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="677.485">526(7571):55</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="356.427" y="677.485">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="360.056" y="677.485">61.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.723" y="686.443">10. Charpentier E, Richter H, van der Oost J, White MF. Biogenesis pathways of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="695.457">RNA guides in archaeal and bacterial CRISPR-Cas adaptive immunity. FEMS</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="704.471">Microbiol Rev. 2015;39(3):428</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="402.122" y="704.471">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="405.693" y="704.471">41.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.723" y="713.485">11. Makarova KS, Wolf YI, Alkhnbashi OS, Costa F, Shah SA, Saunders SJ,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="722.443">Barrangou R, Brouns SJ, Charpentier E, Haft DH, et al. An updated</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="731.457">evolutionary classification of CRISPR-Cas systems. Nat Rev Microbiol. 2015;</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.746" y="740.471">13(11):722</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="348.49" y="740.471">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="352.061" y="740.471">36.</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.139">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.139">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.139">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="491.979" y="48.139">Page 13 of 14</span>
   </p>
   <img src="./images/page.13.1.png" width="50%"/>
   <span x="304.724" y="445.216" style="font-size:10pt;" class="image">image.13.1[131*10]</span>
  </div>
  <div class="page">
   <div class="running"/>
   <p>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="104.492">12. Koonin EV, Makarova KS, Zhang F. Diversity, classification and evolution of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="113.449">CRISPR-Cas systems. Curr Opin Microbiol. 2017;37:67</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="220.309" y="113.449">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="223.881" y="113.449">78.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="122.746">13. Liu T, Liu Z, Ye Q, Pan S, Wang X, Li Y, Peng W, Liang Y, She Q, Peng N.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="131.76">Coupling transcriptional activation of CRISPR-Cas system and DNA repair genes</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="140.718">by Csa3a in Sulfolobus islandicus. Nucleic Acids Res. 2017;45(15):8978</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="261.978" y="140.718">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="265.437" y="140.718">92.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="150.015">14. Makarova KS, Grishin NV, Shabalina SA, Wolf YI, Koonin EV. A putative RNA-</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="159.029">interference-based immune system in prokaryotes: computational analysis</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="167.986">of the predicted enzymatic machinery, functional analogies with eukaryotic</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="177.001">RNAi, and hypothetical mechanisms of action. Biol Direct. 2006;1:7.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="186.298">15. Lemak S, Nocek B, Beloglazova N, Skarina T, Flick R, Brown G, Joachimiak A,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="195.255">Savchenko A, Yakunin AF. The CRISPR-associated Cas4 protein Pcal_0546</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="204.269">from Pyrobaculum calidifontis contains a [2Fe-2S] cluster: crystal structure</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="213.284">and nuclease activity. Nucleic Acids Res. 2014;42(17):11144</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="238.394" y="213.284">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="241.966" y="213.284">55.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="222.581">16. Zhang J, Kasciukovic T, White MF. The CRISPR associated protein Cas4 is a 5</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="280.007" y="222.581">′</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="283.069" y="222.581">to</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="75.005" y="231.538">′</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="78.123" y="231.538">DNA exonuclease with an iron-sulfur cluster. PLoS One. 2012;7(10):e47232.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="240.836">17. Krajewski WW, Fu X, Wilkinson M, Cronin NB, Dillingham MS, Wigley DB.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="249.85">Structural basis for translocation by AddAB helicase-nuclease and its arrest</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="258.807">at chi sites. Nature. 2014;508(7496):416</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="181.531" y="258.807">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="185.046" y="258.807">9.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="268.105">18. Wilkinson M, Chaban Y, Wigley DB. Mechanism for nuclease regulation in</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="277.119">RecBCD. elife. 2016;5</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="286.36">19. Wigley DB. Bacterial DNA repair: recent insights into the mechanism of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="295.374">RecBCD</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:7.5;font-style:normal;font-weight:normal;" x="93.94" y="295.374">,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="97.512" y="295.374">AddAB and AdnAB.</span>
    <span style="fill:#131413;font-family:ddaecc+cmlsrhadvtt1b53b5fb.i;font-size:7.5;font-style:normal;font-weight:normal;" x="154.999" y="295.374">Nat Rev Microbiol</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="203.754" y="295.374">. 2013;11(1):9</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="240.435" y="295.374">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="244.007" y="295.374">13.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="304.671">20. Zhou C, Pourmal S, Pavletich NP: Dna2 nuclease-helicase structure,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="313.628">mechanism and regulation by Rpa. Elife. 2015;4. doi:10.7554/eLife.09832.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="322.926">21. Steczkiewicz K, Muszewska A, Knizewski L, Rychlewski L, Ginalski K.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="331.94">Sequence, structure and functional diversity of PD-(D/E)XK</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="340.897">phosphodiesterase superfamily. Nucleic Acids Res. 2012;40(15):7016</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="263.623" y="340.897">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="267.195" y="340.897">45.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="350.195">22. Hooton SP, Brathwaite KJ, Connerton IF. The Bacteriophage carrier state of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="359.209">campylobacter jejuni features changes in host non-coding RNAs and the</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="368.166">Acquisition of new Host-Derived CRISPR spacer sequences. Front Microbiol.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="377.18">2016;7:355.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="386.478">23. Hooton SP, Connerton IF. Campylobacter jejuni acquire new host-derived</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="395.435">CRISPR spacers when in association with bacteriophages harboring a</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="404.449">CRISPR-like Cas4 protein. Front Microbiol. 2014;5:744.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="413.747">24. Krupovic M, Cvirkaite-Krupovic V, Prangishvili D, Koonin EV. Evolution of an</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="422.761">archaeal virus nucleocapsid protein from the CRISPR-associated Cas4</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="431.718">nuclease. Biol Direct. 2015;10:65.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="441.016">25. Krupovic M, Beguin P, Koonin EV: Casposons: the mobile elements that</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="450.03">gave rise to the adaptation module of CRISPR-Cas systems. Curr Opin</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="458.987">Microbiol 2017, in press.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="468.285">26. Krupovic M, Makarova KS, Forterre P, Prangishvili D, Koonin EV. Casposons: a</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="477.299">new superfamily of self-synthesizing DNA transposons at the origin of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="486.256">prokaryotic CRISPR-Cas immunity. BMC Biol. 2014;12:36.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="495.554">27. Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="504.568">Gapped BLAST and PSI-BLAST: a new generation of protein database search</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="513.525">programs. Nucleic Acids Res. 1997;25(17):3389</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="202.508" y="513.525">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="206.079" y="513.525">402.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="522.822">28. Aziz RK, Devoid S, Disz T, Edwards RA, Henry CS, Olsen GJ, Olson R,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="531.837">Overbeek R, Parrello B, Pusch GD, et al. SEED servers: high-performance</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="540.851">access to the SEED genomes, annotations, and metabolic models. PLoS</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="549.808">One. 2012;7(10):e48053.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.693" y="559.105">29. Makarova KS, Koonin EV. Annotation and classification of CRISPR-Cas</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="568.12">systems. Methods Mol Biol. 2015;1311:47</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="187.54" y="568.12">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="191.055" y="568.12">75.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.692" y="577.36">30. Silas S, Makarova KS, Shmakov S, Páez-Espino D, Mohr G, Liu Y, Davison M,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="586.374">Roux S, Krishnamurthy SR, Hansen LL et al: On the origin of reverse</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="595.388">transcriptase-using CRISPR-Cas systems and their hyper-diverse, enigmatic</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="604.346">spacer repertoires. MBio, in press 2017.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.692" y="613.644">31. Rinke C, Schwientek P, Sczyrba A, Ivanova NN, Anderson IJ, Cheng JF, Darling A,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="622.658">Malfatti S, Swan BK, Gies EA, et al. Insights into the phylogeny and coding</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.716" y="631.615">potential of microbial dark matter. Nature. 2013;499(7459):431</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="235.955" y="631.615">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="239.357" y="631.615">7.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.691" y="640.913">32. Castelle CJ, Wrighton KC, Thomas BC, Hug LA, Brown CT, Wilkins MJ,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="649.927">Frischkorn KR, Tringe SG, Singh A, Markillie LM, et al. Genomic expansion of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="658.941">domain archaea highlights roles for organisms from new phyla in anaerobic</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="667.898">carbon cycling. Curr Biol. 2015;25(6):690</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="184.761" y="667.898">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="188.333" y="667.898">701.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.691" y="677.196">33. Shmakov S, Abudayyeh OO, Makarova KS, Wolf YI, Gootenberg JS,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="686.21">Semenova E, Minakhin L, Joung J, Konermann S, Severinov K, et al.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="695.168">Discovery and functional characterization of diverse class 2 CRISPR-Cas</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="704.182">systems. Mol Cell. 2015;60(3):385</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="164.068" y="704.182">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="167.64" y="704.182">97.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="56.691" y="713.48">34. Brown CT, Hug LA, Thomas BC, Sharon I, Castelle CJ, Singh A,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="722.438">Wilkins MJ, Wrighton KC, Williams KH, Banfield JF. Unusual biology</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="731.452">across a group comprising more than 15% of domain bacteria. Nature.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="71.715" y="740.466">2015;523(7559):208</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="127.161" y="740.466">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="130.846" y="104.485">11.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.723" y="104.485">35. Burstein D, Sun CL, Brown CT, Sharon I, Anantharaman K, Probst AJ, Thomas BC,</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="113.442">Banfield JF. Major bacterial lineages are essentially devoid of CRISPR-Cas viral</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="122.463">defence systems. Nat Commun. 2016;7:10613.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="131.477">36. Puigbo P, Makarova KS, Kristensen DM, Wolf YI, Koonin EV. Reconstruction</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="140.491">of the evolution of microbial defense systems. BMC Evol Biol. 2017;17(1):94.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="149.448">37. Kristensen DM, Wolf YI, Koonin EV. ATGC database and ATGC-COGs: an</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="158.463">updated resource for micro- and macro-evolutionary studies of prokaryotic</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="167.477">genomes and protein family annotation. Nucleic Acids Res. 2017;45(D1):D210</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="525.997" y="167.477">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="529.399" y="167.477">8.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="176.491">38. Takeuchi N, Wolf YI, Makarova KS, Koonin EV. Nature and intensity of selection</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="185.448">pressure on CRISPR-associated genes. J Bacteriol. 2012;194(5):1216</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="501.223" y="185.448">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="504.737" y="185.448">25.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="194.462">39. Dillingham MS. Superfamily I helicases as modular components of</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="203.476">DNA-processing machines. Biochem Soc Trans. 2011;39(2):413</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="496.064" y="203.476">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="499.635" y="203.476">23.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="212.49">40. Epshtein V. UvrD helicase: an old dog with a new trick: how one step</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.748" y="221.448">backward leads to many steps forward. BioEssays. 2015;37(1):12</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="500.372" y="221.448">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="503.887" y="221.448">9.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="230.462">41. Galperin MY, Makarova KS, Wolf YI, Koonin EV. Expanded microbial genome</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="239.477">coverage and improved protein family annotation in the COG database.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="248.491">Nucleic Acids Res. 2015;43(Database issue):D261</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="456.491" y="248.491">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="460.006" y="248.491">9.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="257.449">42. Schwarz FW, Toth J, van Aelst K, Cui G, Clausing S, Szczelkun MD, Seidel R.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="266.463">The helicase-like domains of type III restriction enzymes trigger long-range</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="275.477">diffusion along DNA. Science. 2013;340(6130):353</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="459.893" y="275.477">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="463.464" y="275.477">6.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="284.491">43. Butterer A, Pernstich C, Smith RM, Sobott F, Szczelkun MD, Toth J. Type III</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="293.449">restriction endonucleases are heterotrimeric: comprising one helicase-</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="302.463">nuclease subunit and a dimeric methyltransferase that binds only one</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="311.477">specific DNA. Nucleic Acids Res. 2014;42(8):5139</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="456.208" y="311.477">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="459.779" y="311.477">50.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="320.491">44. Gupta YK, Chan SH, Xu SY, Aggarwal AK. Structural basis of asymmetric DNA</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="329.448">methylation and ATP-triggered long-range diffusion by EcoP15I. Nat</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="338.462">Commun. 2015;6:7363.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="347.477">45. Marinus MG, Lobner-Olesen A. DNA Methylation. EcoSal Plus. 2014;6(1). doi:</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="356.491">10.1128/ecosalplus.ESP-0003-2013.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="365.448">46. Iyer LM, Abhiman S, Aravind L. MutL homologs in restriction-modification</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="374.462">systems and the origin of eukaryotic MORC ATPases. Biol Direct. 2008;3:8.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="383.476">47. Aravind L, Walker DR, Koonin EV. Conserved domains in DNA repair proteins</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="392.49">and evolution of repair systems. Nucleic Acids Res. 1999;27(5):1223</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="510.066" y="392.49">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="513.638" y="392.49">42.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="401.447">48. Lenhart JS, Schroeder JW, Walsh BW, Simmons LA. DNA repair and genome</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="410.461">maintenance in Bacillus Subtilis. Microbiol Mol Biol Rev. 2012;76(3):530</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="520.214" y="410.461">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="523.786" y="410.461">64.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="419.476">49. Makarova KS, Wolf YI, Snir S, Koonin EV. Defense islands in bacterial and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="428.49">archaeal genomes and prediction of novel defense systems. J Bacteriol.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="437.447">2011;193(21):6039</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="370.147" y="437.447">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="373.719" y="437.447">56.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="446.461">50. Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="455.475">high throughput. Nucleic Acids Res. 2004;32(5):1792</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="467.83" y="455.475">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="471.344" y="455.475">7.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="464.489">51. Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7:</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="473.446">improvements in performance and usability. Mol Biol Evol. 2013;30(4):772</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="521.574" y="473.446">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="525.033" y="473.446">80.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="482.461">52. Price MN, Dehal PS, Arkin AP. FastTree 2</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="434.154" y="482.461">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="437.726" y="482.461">approximately maximum-</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="491.475">likelihood trees for large alignments. PLoS One. 2010;5(3):e9490.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="500.489">53. Edgar RC. Search and clustering orders of magnitude faster than BLAST.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="509.446">Bioinformatics. 2010;26(19):2460</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="410.229" y="509.446">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="413.801" y="509.446">1.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="518.46">54. Soding J. Protein homology detection by HMM-HMM comparison.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="527.474">Bioinformatics. 2005;21(7):951</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="403.369" y="527.474">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="406.941" y="527.474">60.</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="304.724" y="536.488">55. Soding J, Remmert M, Biegert A, Lupas AN, et al. Nucleic Acids Res. 2006;</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="319.747" y="545.445">34(Web Server issue):W374</span>
    <span style="fill:#131413;font-family:ddaedd+nhkrcqadvttb5929f4c+20;font-size:7.5;font-style:normal;font-weight:normal;" x="396.056" y="545.445">–</span>
    <span style="fill:#131413;font-family:ddadpb+dwgljyadvttb5929f4c;font-size:7.5;font-style:normal;font-weight:normal;" x="399.571" y="545.445">8.</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.699" y="648.8">
     <b>•</b>
    </span>
    <span style="fill:#131413;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.08" y="648.8">We accept pre-submission inquiries</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.698" y="660.8">
     <b>•</b>
    </span>
    <span style="fill:#131413;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.079" y="660.8">Our selector tool helps you to ﬁnd the most relevant journal</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.698" y="672.8">
     <b>•</b>
    </span>
    <span style="fill:#292829;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.079" y="672.8">We provide round the clock customer support</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.698" y="684.8">
     <b>•</b>
    </span>
    <span style="fill:#292829;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.079" y="684.8">Convenient online submission</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.698" y="696.8">
     <b>•</b>
    </span>
    <span style="fill:#131413;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.079" y="696.8">Thorough peer review</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.698" y="708.8">
     <b>•</b>
    </span>
    <span style="fill:#292829;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.079" y="708.8">Inclusion in PubMed and all major indexing services</span>
    <span style="fill:#131413;font-family:lssjsgopensans-extrabold;font-size:7.0;font-style:normal;font-weight:bold;" x="323.698" y="720.8">
     <b>•</b>
    </span>
    <span style="fill:#131413;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="330.079" y="720.8">Maximum visibility for your research</span>
    <span style="fill:#292829;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="323.698" y="737.677">Submit your manuscript at</span>
    <span style="fill:#292829;font-family:vhbytbopensans-light;font-size:7.0;font-style:normal;font-weight:normal;" x="323.698" y="745.678">www.biomedcentral.com/submit</span>
    <span style="fill:#131413;font-family:tqlffqoregonldobook;font-size:11.0;font-style:normal;font-weight:normal;" x="318.398" y="620.867">Submit your next manuscript to BioMed Central</span>
    <span style="fill:#131413;font-family:tqlffqoregonldobook;font-size:11.0;font-style:normal;font-weight:normal;" x="318.398" y="634.066">and we will help you at every step:</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="56.693" y="48.139">Hudaiberdiev</span>
    <span style="fill:#131413;font-family:ddafcb+fgymkfadvtt7329fd89.i;font-size:8.0;font-style:normal;font-weight:normal;" x="103.861" y="48.139">et al. BMC Evolutionary Biology</span>
    <span style="fill:#131413;font-family:myriadpro-regular;font-size:8.0;font-style:normal;font-weight:normal;" x="209.418" y="48.139">(2017) 17:232</span>
    <span style="fill:#131413;font-family:ddadpa+hcfgkdadvtte45e47d2;font-size:8.0;font-style:normal;font-weight:normal;" x="491.979" y="48.139">Page 14 of 14</span>
   </p>
  </div>
 </body>
</html>
